pkg/uptimerobot.go

```diff
@@ -112,6 +112,33 @@ func (a AlertContact) String() string {
 	return render(alertContactTemplate, a)
 }
 
+type Status int
+
+const (
+	Up        Status = 2
+	Down      Status = 9
+	MaybeDown Status = 8
+	Unknown   Status = 1
+	Paused    Status = 0
+)
+
+func (s Status) String() string {
+	statuses := map[Status]string{
```

Comment:
Let's define this map outside the function, similar to other enum-style mappings.